src/screens/WIP/SongInfo.js

```diff
@@ -37,14 +38,41 @@ export function SongInfo() {
     };
   });
 
-  const playSong = useCallback(() => sound.play(), [sound]);
+  const playPause = () => {
+    sound.isPlaying() ? sound.pause() : sound.play();
+  };
+  const jumpPrev15Seconds = () => {
+    jumpSeconds(-10);
+  };
+  const jumpNext15Seconds = () => {
+    jumpSeconds(10);
+  };
+
+  const jumpSeconds = secondsToJump => {
+    sound.getCurrentTime(seconds => {
+      var time = seconds + secondsToJump;
+      if (time < 0) {
+        time = 0;
+      } else if (time > sound.getDuration) {
+        time = sound.getDuration;
+      }
+      sound.setCurrentTime(time);
+    });
+  };
 
   return (
     <SafeAreaView
       style={
         isDarkTheme === true ? stylesDark.containerDark : styles.container
```

Comment:
for it to be cleaner you can do 
```suggestion
        isDarkTheme ? stylesDark.containerDark : styles.container
```